### ISSUE TITLE
Break out data work: Stats possible now, what would require unification of user data across devices

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5899]
Per discussion with [~accountid:60ec9eaaf314ff006a44bb42], [~accountid:634d8e68fe5ff375235781b3] will break down the data work to be done now, and what will await later work (blocked on user data unification across devices).

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-4119)
┆Reporter: Matt Cleinman

**Comment by data-sync-user** [comment]
➤ Matt Cleinman commented:

Closing this out.

From note I sent to  Yeonjoo Yoo - I’ve completed the work we talked about on the spreadsheet: https://docs.google.com/spreadsheets/d/1EdPD4bbSWW4VVfWPcQt4YqSB_Zpecfu1_f_5DKALAdw/edit#gid=0 ( https://docs.google.com/spreadsheets/d/1EdPD4bbSWW4VVfWPcQt4YqSB_Zpecfu1_f_5DKALAdw/edit#gid=0|smart-link ) Data work is now broken out into columns J, K, and L. I’ve added comments on the header cells, but they basically break down as such:
column J: Things that can be done once data is flowing through the app.
column L: Work that requires the ability to link user data across devices, something that is being worked on by other folks. (My understanding is that this is more of a “getting approval” challenge than a technical one.)
column K: Work that isn’t needed - but gets at similar questions as column L, but on a device level (instead of a user level). If it’s going to take a long time to get that user-permission, this may be nice to have in the interim - but I’m not sure we really need to work on this unless Santiago asks for it. (I added this column proactively) 

One thing not included in that spreadsheet is the desired filtering capabilities from the PRD (https://mozilla-hub.atlassian.net/wiki/spaces/SECPRV/pages/117309748/Metrics+Collection+Dashboards+PRD ( https://mozilla-hub.atlassian.net/wiki/spaces/SECPRV/pages/117309748/Metrics+Collection+Dashboards+PRD|smart-link ) ). That section is mostly things that won’t be possible until we can connect user data across devices, but the client OS and client version are things that could be included now.
